featuretools/entityset/entityset.py

```diff
@@ -642,8 +638,6 @@ def add_dataframe(self,
             extra_params = []
             if index is not None:
                 extra_params.append('index')
-            if make_index:
-                extra_params.append('make_index')
```

Comment:
what if the user passes a ww dataframe but wants to use make index?

Reply:
If I'm looking at things right, the user cannot use `make_index` with a ww dataframe (although they can try). If the user passes a ww dataframe, there is a check that the dataframe has an index column set, and if not a ValueError is raised: https://github.com/alteryx/featuretools/blob/bf3147c67667bd91ec81663bdcaa136f4b84c606/featuretools/entityset/entityset.py#L639-L640

It is possible that the user would pass a ww dataframe that already has an index set, and also pass make_index=True, in which case we would ignore the `make_index` parameter. So, I guess it does make sense to leave that check in place for the warning.

Reply:
This means we're losing a behavior that previously existed where a user could init Woodwork on a dataframe before EntitySet init while also creating an index. 

Not that big of a deal to lose it, I think, but just wanted to point out.